hack/local-down-karmada.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The script to remove the karmada started by hack/local-up-karmada.sh
+# example:
+# 1.hack/local-down-kaarmada.sh
+# 2.hack/local-down-kaarmada.sh -y ('-y' mean remove local images)
```

Comment:
I prefer explicit name the arguments。 E.g. 
```
# remove images when tear down.
hack/local-down-kaarmada.sh --remove-images
# not remove images when tear down.
hack/local-down-kaarmada.sh --remove-images=false
```

Reply:
I agree with you. I'll revise it later

Reply:
karmada, not ka`a`rmada

Reply:
@ikaven1024 Thanks for your review. I have changed it to `-r` and provided relevant instructions.

Reply:
> karmada, not ka`a`rmada

@RainbowMango Thanks, it has been moved to usage

> karmada, not ka`a`rmada

@RainbowMango Thanks, it has been moved to usage